commands/sync.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"github.com/bndw/pick/backends"
+	"github.com/bndw/pick/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "sync [other-pick-safe]",
+		Short: "Sync current safe with another pick safe",
+		Long:  "The sync command is used to sync the current pick safe with another pick safe.",
```

Comment:
Makes sense- could you please clarify the behavior in a function comment so it's explicitly documented?

Related: https://github.com/bndw/pick/issues/123